features/config-get-field.feature

```diff
@@ -0,0 +1,84 @@
+Feature: Get the value of a constant or global defined in wp-config.php file
+
+  Background:
+    Given a WP install
+
+  Scenario: Get the value of an existing wp-config.php constant
+    When I try `wp config get --constant=DB_NAME`
+    And STDOUT should be:
```

Comment:
Use `Then` instead of `And` here, to provide for a better reading experience. So, it should be `When` -> `Then` instead of `When` -> `And`.